vars/getComponentConfigs.groovy

```diff
@@ -1,14 +1,27 @@
 #!/usr/bin/groovy
+
 import groovy.json.JsonSlurperClassic
 
-def call(componentConfigs, configGitRepo, configGitRef) {
+def call(componentConfigs, configGitRepo, configGitRef, tryMasterOnFail = false) {
```

Comment:
New optional argument, if true will use the master branch to retrieve the component configs if the configGitRef passed in does not exist.